main.css

```diff
@@ -1,3 +1,83 @@
-body {
-  background-color: aliceblue;
+nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 12px;
+}
+
+nav > p {
+  left: 12px;
+  width: 90px;
+  height: 28px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 28px;
+  color: #28352f;
+}
+
+nav > a {
+  margin: auto 0;
+}
+
+.headline {
+  background-image: url(img/header-illsutration-mobile.svg);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+header > h2 {
+  width: 327px;
+  left: 24px;
+  top: 64px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 58px 0;
+}
+
+header > h2 > span {
+  color: #36b37e;
+}
+
+.desc {
+  width: 327px;
+  height: 164px;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #172b4d;
+}
```

Comment:
- [ ] I noticed that there is an extra margin at the top and bottom of the paragraph text and I inspected it on developer tools and found out that the `p` tag has a default margin. I suggest you give this paragraph class a `margin-top: 0` and `margin-bottom: 0` so that you will have the same margin as that of your chosen template.